schema/policy_test.go

```diff
@@ -510,18 +510,22 @@ var _ = Describe("Policies", func() {
 				Expect(p).To(BeNil())
 			}
 
-			It("should fail to create when invalid value type is provided", func() {
-				givenPolicyWithScope(map[string]interface{}{})
-				thenCreationShouldFail("\"scope\" should be a string, or a list")
-			})
+			DescribeTable("should fail to create when invalid value type is provided",
+				func(scope interface{}) {
+					givenPolicyWithScope(scope)
+					thenCreationShouldFail("\"scope\" should be a list of strings")
+				},
+				Entry("dictionary", map[string]interface{}{}),
+				Entry("string", "admin"),
+			)
 
 			It("should fail to create when non-string is provided in scope list", func() {
 				givenPolicyWithScope([]interface{}{"admin", 123})
 				thenCreationShouldFail("Token type at position 1 in scope list should be a string")
 			})
 
-			It("should fail to create when invalid token type is provided", func() {
-				givenPolicyWithScope("flower")
+			It("should fail to create when a string is provided", func() {
```

Comment:
fail because of a string (not list of strings) is covered in the second `Entry` of  `should fail to create when invalid value type is provided`. here we're checking if creation fails because of an invalid _value_